### DISCUSSION TITLE
HxAutosuggest Gets Hung Up and I am not sure why.

### DISCUSSION THREAD

**Discussion by chobo2** [havit/Havit.Blazor#691] [Q&A] [answered]
Hi

I can't recreate my problem in a sample application at this point but I am wondering if anyone else is having issues where if you say type "fkakfadkfk" and then backspace and type "Jim" that the auto suggest dropdwon box just does not show up. I have to reclick into the box to get it to reappear so the data seems to be there, just not displaying properly.

It only randomly happens, so I am having a hard time to figure out what the cause is. Wondering if anyone has run into this and can point me to some solutions.

**Comment by hakenr** [comment, marked as answer]
Hello @chobo2,
was this related to #702 or is it another issue? (In such case please file an issue with some repro steps.)
Thank you.

**Reply by chobo2** [reply]
Right forgot I opened a discussion as was not sure if it was my problem or something wrong. This is the same issue as 702.
